### ISSUE TITLE
AVPlayerViewController UI with WistiaPlayer with Stats? 

### ISSUE THREAD

**Issue by ADWilliams** [wistia/WistiaKit#88]
Hey,

I'm trying to use `WistiaPlayer` with a standard `AVPlayerViewController` like so
```Swift
   private(set) lazy var playerViewController: AVPlayerViewController = {
        let viewcontroller = AVPlayerViewController()
        let wistaPlayer = WistiaPlayer(referrer: "", requireHLS: true)
        wistaPlayer.configure(with: viewcontroller)
        return viewcontroller
    }()
```

I've realized that this isn't recording stats, which makes sense since it's using the underlying `AVPlayer` object. Is there a way to use the stock `AVPLayerViewController` while still keeping the video stats? If not I guess this is a feature request 😅.

Thanks.

**Comment by spinosa** [comment]
`WistiaPlayer` creates an underlying `AVPlayer` and registers observers on it (and it's `playerItem`s) which allow the `WistiaPlayer` to report stats.  

When you `.configure(with: viewcontroller)`, the `WistiaPlayer`'s `AVPlayer` is set on the given `AVPlayerViewController`.

The setup you describe should report video stats.

**Comment by ADWilliams** [comment]
After digging around a bit I noticed that the events are being sent, but not showing up in my dashboard. I also noticed that we were calling the play and pause functions on the `AVPlayerViewController.player` property.  After keeping a reference to the original `WistiaPlayer` object and calling `play()` and `pause()` there instead caused the stats to show up properly. 

Hopefully that sheds some light on what's going on. It's working well for me now. Thanks!
